README.md

```diff
@@ -1,38 +1,66 @@
+  [![Made with reactive-vscode](https://img.shields.io/badge/made_with-reactive--vscode-212121?style=flat&labelColor=fff&logo=visual-studio-code&logoColor=fff)](https://kermanx.github.io/reactive-vscode/)
+
+  [**🐛 Report Bug**](https://github.com/yunsii/vscode-links/issues/new) &nbsp;&nbsp;•&nbsp;&nbsp;
+  [**✨ Request Feature**](https://github.com/yunsii/vscode-links/issues/new) &nbsp;&nbsp;•&nbsp;&nbsp;
+  [**🔍 View Source**](https://github.com/yunsii/vscode-links)
+
+</div>
+
+---
+
+## Table of Contents
+
+- [Features](#features)
+- [Usage](#usage)
+- [Roadmap](#roadmap)
+- [License](#license)
+
+## Features
+
+- Initialize and manage project-related links directly from VS Code.
+- Quick access to project links via the Command Palette.
+- Supports custom link resources.
 
 ## Usage
 
-1. Initialize current project related links resources like:
+1. Add your project links to `.vscode/settings.json`:
 
-```jsonc
-// .vscode/settings.json
-{
-  "links.resources": [
-    {
-      "url": "https://github.com/yunsii/vscode-links",
-      "title": "GitHub Repo"
-    }
-    // ...
-  ]
-}
-```
+   ```jsonc
+   {
+     "links.resources": [
+       {
+         "url": "https://github.com/yunsii/vscode-links",
+         "title": "GitHub Repository"
+       }
+       // Add more links as needed
+     ]
+   }
+   ```
 
-2. Open the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette)
+2. Open the [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) (<kbd>Ctrl+Shift+P</kbd> or <kbd>Cmd+Shift+P</kbd> on **macOS**).
 
-3. Select **Links Open ...**
+3. Type **Links: Open...** and select it.
 
-4. Select the link you want to open by default browser 🎉
+4. Choose the link you want to open in your default browser 🎉.
 
 ## Roadmap
 
-- [ ] Support auto parse git repo web links
+- [ ] Support auto-parsing of git repository web links
   - [ ] GitHub
   - [x] CODING
+- [ ] Localization support for multiple languages
+- [ ] Improve link management UI
```

Comment:
I think it should be "Improve links management UI"?

Reply:
Fixed 🫡